test/yargs.mjs

```diff
@@ -663,38 +645,36 @@ describe('yargs dsl tests', () => {
 
   describe('locale', () => {
     function loadLocale(locale) {
-      clearRequireCache();
-      yargs = require('../index.cjs');
       process.env.LC_ALL = locale;
     }
 
     it('uses english as a default locale', () => {
       ['LANGUAGE', 'LC_ALL', 'LANG', 'LC_MESSAGES'].forEach(e => {
         delete process.env[e];
       });
-      yargs.locale().should.equal('en_US');
+      yargs().locale().should.equal('en_US');
     });
 
     it("detects the operating system's locale", () => {
       loadLocale('es_ES.UTF-8');
-      yargs.locale().should.equal('es_ES');
+      yargs().locale().should.equal('es_ES');
       loadLocale('en_US.UTF-8');
     });
 
     it("should not detect the OS locale if detectLocale is 'false'", () => {
       loadLocale('es_ES.UTF-8');
+      const y = yargs()
+        .command('blerg', 'blerg command')
+        .help('h')
+        .wrap(null)
+        .detectLocale(false);
 
       const r = checkOutput(() => {
-        yargs(['snuh', '-h'])
-          .command('blerg', 'blerg command')
-          .help('h')
-          .wrap(null)
-          .detectLocale(false)
-          .parse();
+        y.parse(['snuh', '-h']);
       });
 
-      yargs.locale().should.equal('en');
-      yargs.getDetectLocale().should.equal(false);
+      y.locale().should.equal('en_US');
```

Comment:
Very minor. Do you know the locale changed from `en` to `en_US`? I wasn't expecting a functional change.

Reply:
y18n is a singleton, and I found an edge-case in tests during my refactor where the state of the locale from a prior test would bleed over. To fix the problem I started calling `this.#shim.y18n.setLocale(DEFAULT_LOCALE);` which sets the locale to `en_US`. I chose `en_US` because I believe it's more correct than just `en` without a country code.

Reply:
👍 Thanks for explanation.